spec/services/distribution_create_service_spec.rb

```diff
@@ -58,6 +58,16 @@
       end
     end
 
+    context "when there's multiple line items and one has insufficient inventory" do
+      let(:too_much_params) { { organization_id: @organization.id, partner_id: @partner.id, storage_location_id: storage_location.id, line_items_attributes: { "0": { item_id: storage_location.items.first.id, quantity: 2 }, "1": { item_id: storage_location.items.last.id, quantity: 500 } } } }
```

Comment:
This is a bit of a preference for me but could you put this into seperate lines like:
```
let(:something) do
   {
      organization_id: ...,
      ...
   }
end
```

I find this a bit easier to read :)